lib/xcodeproj/constants.rb

```diff
@@ -104,6 +104,7 @@ module Constants
       :dynamic_library   => 'com.apple.product-type.library.dynamic',
       :static_library    => 'com.apple.product-type.library.static',
       :bundle            => 'com.apple.product-type.bundle',
+      :octest_bundle     => 'com.apple.product-type.bundle',
```

Comment:
Note that it could be worth converting all the fixtures to XCTest someday (but that's for another PR)